Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/DefaultPointerMediator.cs

```diff
@@ -182,27 +187,31 @@ public virtual void UpdatePointers()
 
         private void ApplyCustomPointerBehaviors()
         {
-            if (CoreServices.InputSystem.FocusProvider is FocusProvider focusProvider)
+            if (pointerPreferences != null)
             {
-                Action<IMixedRealityPointer, PointerBehavior> setPointerState =
-                    (ptr, behavior) =>
-                    {
-                        if (behavior == PointerBehavior.Default)
+                if (CoreServices.InputSystem.FocusProvider is FocusProvider focusProvider)
```

Comment:
You can remove this if check, I believe it was there unnecessarily. We don't do anything with the focus provider. I should have deleted that when I reworked the code to use pointer prefs, but I forgot.

Reply:
ok, removed it